docs/guide/validation.md

```diff
@@ -8,76 +8,162 @@ sidebar_position: 5
 
 Fiber can make _great_ use of the validator package to ensure correct validation of data to store.
 
-* [Official validator Github page \(Installation, use, examples..\).](https://github.com/go-playground/validator)
+- [Official validator Github page \(Installation, use, examples..\).](https://github.com/go-playground/validator)
 
 You can find the detailed descriptions of the _validations_ used in the fields contained on the structs below:
 
-* [Detailed docs](https://pkg.go.dev/github.com/go-playground/validator?tab=doc)
+- [Detailed docs](https://pkg.go.dev/github.com/go-playground/validator?tab=doc)
 
 ```go title="Validation Example"
-type Job struct{
-    Type          string `validate:"required,min=3,max=32"`
-    Salary        int    `validate:"required,number"`
-}
+package main
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/go-playground/validator/v10"
+	"github.com/gofiber/fiber/v2"
+)
+
+type (
+	User struct {
+		Name string `validate:"required,min=5,max=20"` // Required field, min 5 char long max 20
+		Age  int    `validate:"required,teener"`       // Required field, and client needs to implement our 'teener' tag format which we'll see later
+	}
+
+	ErrorResponse struct {
+		Error       bool
+		FailedField string
+		Tag         string
+		Value       interface{}
+	}
+
+	XValidator struct {
+		validator *validator.Validate
+	}
+
+	GlobalErrorHandlerResp struct {
+		Success bool   `json:"success"`
+		Message string `json:"message"`
+	}
+)
+
+// This is the validator instance
+// for more information see: https://github.com/go-playground/validator
+var validate = validator.New()
 
-type User struct{
-    Name          string  `validate:"required,min=3,max=32"`
-    // use `*bool` here otherwise the validation will fail for `false` values
-    // Ref: https://github.com/go-playground/validator/issues/319#issuecomment-339222389
-    IsActive      *bool   `validate:"required"`
-    Email         string  `validate:"required,email,min=6,max=32"`
-    Job           Job     `validate:"dive"`
-}
+func (v XValidator) Validate(data interface{}) []ErrorResponse {
+	Error := []ErrorResponse{}
```

Comment:
Can you make this var's name validationErrors

Reply:
Oke, completed 👍